guides/tutorial/complex-arguments.md

```diff
@@ -137,7 +137,7 @@ Everyone else gets an `"Access denied"` error for this field.
 
 Here's our mutation in action in GraphiQL.
 
-<img style="box-shadow: 0 0 6px #ccc;" src="assets/tutorial/graphiql_create_user.png" alt=""/>
+<img style="box-shadow: 0 0 6px #ccc;" src="./assets/tutorial/graphiql_create_user.png" alt=""/>
```

Comment:
I'm pretty sure that this is correct when viewed as the generated HTML docs. Make sure you're testing this by running `mix hex.docs` and then opening `./docs/index.html` instead of using your editor's markdown viewer.